pypowsybl/security.py

```diff
@@ -77,14 +80,43 @@ def get_table(self):
                                limit_violation.side.name])
         return table
 
+    def get_branch_results(self):
+        return create_data_frame_from_series_array(_pypowsybl.get_branch_results(self.ptr))
+
+    def get_bus_results(self):
+        return create_data_frame_from_series_array(_pypowsybl.get_bus_results(self.ptr))
+
+    def get_three_windings_transformer_results(self):
+        return create_data_frame_from_series_array(_pypowsybl.get_three_windings_transformer_results(self.ptr))
+
 
 class SecurityAnalysis(ContingencyContainer):
     def __init__(self, ptr):
         ContingencyContainer.__init__(self, ptr)
 
-    def run_ac(self, network: Network, parameters: Parameters = Parameters(), provider='OpenSecurityAnalysis') -> SecurityAnalysisResult:
+    def run_ac(self, network: Network, parameters: Parameters = Parameters(),
+               provider='OpenSecurityAnalysis') -> SecurityAnalysisResult:
         return SecurityAnalysisResult(_pypowsybl.run_security_analysis(self.ptr, network.ptr, parameters, provider))
 
+    def add_state_monitors(self, contingency_context_type: _pypowsybl.ContingencyContextType,
```

Comment:
To be renamed to `add_monitored_elements`, and "convenient" methods to be added for pre/post contingency(see previous comment in the pull request).